dataset/test/bin_test.cpp

```diff
@@ -219,7 +219,7 @@ TEST_P(BinTest, rebin_2d_with_2d_coord) {
   auto table = GetParam();
   auto xy = bin(table, {edges_x_coarse, edges_y_coarse});
   Variable edges_y_2d = makeVariable<double>(Dims{Dim::X, Dim::Y}, Shape{2, 3},
-                                             Values{-2, 1, 2, -1, 0, 3});
+                                             Values{-2, 1, 2, -3, 0, 3});
```

Comment:
Does this change mean anything?

Reply:
Yes and now: This test uses a hacky way of adding 2-D edges, which resulted in events being outside the edges. Depending on whether also the out dim was rebinned, `bin` would or would not detect that and drop those events.

Reply:
Gotcha